### DISCUSSION TITLE
What files do i copy to transfer my settings when doing a fresh install?

### DISCUSSION THREAD

**Discussion by djwashout** [TheLastBen/fast-stable-diffusion#2805] [General]
I thought config.json and ui-config.json from the main stable-diffusion-webui dir would allow me to preserve my settings when doing a new git pull but it does not. Wondering where the settings are actually stored.

**Comment by TheLastBen** [comment]
which platform ? colab, runpod or paperspace ?

**Reply by djwashout** [reply]
ohh sorry. colab.

**Comment by djwashout** [comment]
still wanting to know what files/dir i need to save to backup the UI settings on google colab. Thanks.

**Comment by djwashout** [comment]
wanting to do a fresh clone, hoping someone can point out the process for transferring over my settings. Every time i do a fresh clone its takes me hours to get everything set up...

**Comment by djwashout** [comment]
well i replaced config.json and ui-config.json with the files from my previous install and it seemed to work! Last time i tried this it did not work.
Glad its resolved. Wish someone would have said so. ohh well.

**Reply by Gelazer** [reply]
You replaced only those two files, after installing a fresh clone and everything was as it was? You mean in terms of just settings? I mean - I recon not all your installed extensions and what have you were back right?

**Reply by djwashout** [reply]
everything was as it was. settings, folders, ect. But YES you have to reinstall extensions.
